parts/k8s/cloud-init/artifacts/chrony.conf

```diff
@@ -0,0 +1,43 @@
+# Use Microsoft NTP servers:
+server time.windows.com
+
+# Ignore stratum in source selection.
+stratumweight 0
+
+# Record the rate at which the system clock gains/losses time.
+driftfile /var/lib/chrony/drift
+
+# Enable kernel RTC synchronization.
+rtcsync
+rtconutc
+
+# In first three updates step the system clock instead of slew
+# if the adjustment is larger than 10 seconds.
+makestep 10 3
```

Comment:
Might want to make this smaller... e.g. 1 second, otherwise (i'm not familiar with chrony specifically) the clock may never sync up.